lambda/src/main/resources/db/migration/V145__add_file_uuid_to_fileproperty.sql

```diff
@@ -0,0 +1,8 @@
+-- Add 'UUID' to FileProperty table
+-- 'UUID' refers to the 'FileID'
+
+INSERT INTO "FileProperty" ("Name", "Description", "FullName", "PropertyType", "Datatype", "Editable", "MultiValue",
```

Comment:
Maybe add a value to the `Description` field to explain what this field is, ie: `FiledID value for the record`

Reply:
I've now added a description. I've also tested the migration locally.